File: cypress/cypress/integration/cypress_poc/pages/your_information.page.js
export const Informacoes = {

  preencher_infos() {
    cy.get('[data-test=firstName]').type('Rod')
    cy.get('[data-test=lastName]').type('Matola')
    cy.get('[data-test=postalCode]').type('000000')
    cy.get('.btn_primary').click()
  }

}

File: selenium/pages/your_information.page.rb
class Informacoes

  def self.preencher_infos
    $driver.find_element(:id, 'first-name').send_keys('Rod')
    $driver.find_element(:id, 'last-name').send_keys('Matola')
    $driver.find_element(:id, 'postal-code').send_keys('000000')
    $driver.find_element(:class, 'btn_primary').click
  end

end
